synergy-design-system/synergy-design-system
# feat: ✨ add support for folder dnd in syn-file

- enable folder dropping
- change button and drop-area text according to webkitdirectory
- added story for folder drop



# Pull Request

## 📖 Description


Adds dnd support for folders

### 🎫 Issues

#583 


## 👩‍💻 Reviewer Notes


Have a look at the new "Directory" story

## 📑 Test Plan


1. set webkitdirectory
2. set accept
3. drag a folder with accept-files and not accept files onto the drop area
4. the number of files should match the number of proper accept files

## ✅ DoD
